src/main/java/com/superdevs/HealthOMeter/entity/Contact.java

```diff
@@ -0,0 +1,225 @@
+package com.superdevs.HealthOMeter.entity;
+
+import javax.persistence.*;
+import javax.validation.constraints.Email;
+import java.time.LocalDateTime;
+import java.util.Set;
+
+@Entity(name = "contact")
+public class Contact {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id")
+    private long id;
+
+    @OneToMany(fetch = FetchType.LAZY, mappedBy = "contact")
+    private Set<BioMetricData> bioMetricData;
+
```

Comment:
Rozmawilismy ze wszystkie obiekty z najnowszym updatem beda featchowane Eager. Po za tym beda w relacji 1:1. Jeden uzytkownik bedzie miec zawsze tylko 1 najbardziej akyualne dane. Jesli je updejtuje to nowe wpisy zastepuja stare a stare trafiaja do HistoricalDataXXX gdzie bedzie relacja 1 kontakt : wiele wpisow historycznych. Chodzi o to zeby do aktualnych danych byl bardzo latwy i szybki dostep bo zakladamy ze UI najczesciej z nich bedzie korzystac.